itests/util/src/main/java/org/apache/hadoop/hive/ql/QTestMetaStoreHandler.java

```diff
@@ -98,14 +98,12 @@ private String getDbTypeConfString() {// "ORACLE", "MYSQL", "MSSQL", "POSTGRES"
   }
 
   public void beforeTest() throws Exception {
-    getRule().before();
```

Comment:
That method is used in other places. And at least at one place that doesn't run in the precommit tests suite. For example, I'm pretty sure `TestSchemaToolForMetastore` doesn't run with precommits. I'm not sure about the others.

Reply:
beforeTest method is only used in CliAdapter class, so I don't understand your concerns here.